scripts/checkdeps

```diff
@@ -75,64 +69,35 @@ done
 if [ -n "$need" ]; then
   echo "**** Your system lacks the following tools needed to $1 $DISTRONAME ****"
   echo $need
+  echo "**** You seem to use a $DISTRO system ****"
 
-  if  [ -f /etc/lsb-release ]; then
-    DISTRO=$(grep DISTRIB_ID /etc/lsb-release | cut -d "=" -f 2)
-  fi
-
-  if [ "$DISTRO" = "Ubuntu" ]; then
-    echo "**** You seem to use a $DISTRO system ****"
-    read -p "would you like to install the needed tools ? (y/n) " ans
-    if ( ! [ -z "`which aptitude 2>/dev/null`" ] && [ "$ans" = "y" ]); then
-      sudo aptitude install $need_pkg
-    elif [ "$ans" = "y" ]; then
-      echo "**** The recommended Package-Manager Aptitude is not installed"
-      read -p "(y) install Aptitude :: (n) use APT " ans
+  case "$DISTRO" in
+    ubuntu|debian)
+      read -p "would you like to install the needed tools ? (y/n) " ans
       if [ "$ans" = "y" ]; then
-        sudo apt-get -y install aptitude
-        sudo aptitude install $need_pkg
-      elif [ "$ans" = "n" ]; then
         sudo apt-get install $need_pkg
       else
         echo "********"
         exit 1
       fi
-    else
-      echo "********"
-      exit 1
-    fi
-  elif ( ! [ -z "`which aptitude 2>/dev/null`" ]); then
-   DISTRO="a Debian" 
-   echo "**** You seem to use $DISTRO system ****"
-    read -p "would you like to install the needed tools ? (y/n) " ans
-    if ([ "$ans" = "y" ] && ! [ $(sudo -n uptime 2>&1|grep "load"|wc -l) ]); then
-      sudo aptitude install $need_pkg
-    elif [ "$ans" = "y" ]; then
-      su -c "aptitude install $need_pkg"
-    else
-      echo "********"
+      ;;
+    fedora|centos|rhel)
+      read -p "would you like to install the needed tools ? (y/n) " ans
+      [ "$ans" = "y" ] && sudo yum install $need_pkg
```

Comment:
Might need to check for yum as I don't have it on my machine, dnf is the replacement on newer versions